packages/web/src/pages/chat-page/ChatPage.tsx

```diff
@@ -15,8 +15,8 @@ const messages = {
   pageDescription: '',
   headerText: 'Messages'
 }
-export const ChatPage = () => {
-  const chatId = useSelector(chatSelectors.getCurrentChatId)
+export const ChatPage = ({ match }: RouteComponentProps<{ id?: string }>) => {
```

Comment:
not really sure why we don't follow this pattern and instead re-parse in other places?